lib/consult/template.rb

```diff
@@ -76,20 +80,34 @@ def ordered_locations
 
     # Concatenate all the source templates together, in the order provided
     def contents
-      ordered_locations.map do |location|
+      @_contents ||= ordered_locations.map do |location|
         location.to_s.start_with?('consul') ? consul_contents(location) : disk_contents(location)
-      end.join
+      end.compact.join
     end
 
     def consul_contents(location)
       [@config[location]].compact.flatten.map do |key|
-        Diplomat::Kv.get(key, options: {}, not_found: :return, found: :return).force_encoding 'utf-8'
+        Diplomat::Kv.get(key, {}, :reject, :return).force_encoding 'utf-8'
+      rescue Diplomat::KeyNotFound
+        if @config[:skip_missing_template]
+          STDERR.puts "Consult: Skipping missing template: #{name}"
```

Comment:
```suggestion
          STDERR.puts "Consult: Skipping missing key: #{name}"
```

?

Reply:
Hm, currently the thing being skipped is the _template_. Maybe it'd be better to include the file path or consul key causing the skip, so you know what's missing?